Makefile

```diff
@@ -40,8 +40,6 @@ plugin: configure
 .PHONY: _travis
 _travis:
 	TEST_LLDB_BINARY="$(TEST_LLDB_BINARY)" \
-	TEST_LLNODE_DEBUG=true \
-	LLNODE_DEBUG=true \
```

Comment:
@joyeecheung I removed those lines because I think this might be affecting performance on OS X machines when generating a core file:

![image](https://user-images.githubusercontent.com/4048656/39647168-3660f17a-4fb4-11e8-926a-dd3107593688.png)

Besides, we have too much noise today with those flags on, especially with fallback symbols and "Not object and not smi", making it harder to read the log. If you want I can revert these changes though.

Reply:
@mmarchini No problems, I think we can enable them back in PRs that need them for debugging purposes.